library/src/main/java/com/wultra/android/mtokensdk/operation/IOperationsService.kt

```diff
@@ -103,10 +103,10 @@ interface IOperationsService {
      * If operations are already polling, this call is ignored
      * and the polling interval won't be changed.
      *
-     * @param pollingInterval Polling interval in milliseconds
+     * @param pollingInterval Polling interval in milliseconds, default value is 7s
      * @param delayStart When true, polling starts after the first [pollingInterval] time passes
      */
-    fun startPollingOperations(pollingInterval: Long, delayStart: Boolean)
+    fun startPollingOperations(pollingInterval: Long = 7_000, delayStart: Boolean)
```

Comment:
Let's add a default (`true`) for the `delatStart` parameter as well, having only the first one as optional is uncomfortable for the usage -> Results in `service.startPollingOperations(delayStart = false)` code.